packages/dcos-integration-test/extra/conftest.py

```diff
@@ -34,13 +34,11 @@ def pytest_collection_modifyitems(session, config, items):
 @pytest.fixture
 def vip_apps(dcos_api_session):
     vip1 = '6.6.6.1:6661'
-    test_app1, _ = get_test_app()
-    test_app1['portDefinitions'][0]['labels'] = {
-        'VIP_0': vip1}
-    test_app2, _ = get_test_app()
-    test_app2['portDefinitions'][0]['labels'] = {
-        'VIP_0': 'foobarbaz:5432'}
-    vip2 = 'foobarbaz.marathon.l4lb.thisdcos.directory:5432'
+    test_app1, _ = get_test_app(vip=vip1)
+    name = 'myvipapp'
+    port = 5432
+    test_app2, _ = get_test_app(vip='{}:{}'.format(name, port))
```

Comment:
Good arguments. First off, the comment was non-blocker.  I wanted point out since we were using formatted strings.

> This is not a matter of practice. 

Consider this: I think, it just out of practice that anything you see in mergebot/mergebot repo mostly has `{named}`.format().  Even for single arguments. Some code-base just avoid empty `{}`, because named format is considered better. That's just the evolution of how python string formats came about. {0}, {} were predecessors of {named}.

Choosing the names for keywords has been straightforward. Just use the same name. 
It does not feel very odd.  The idea is we read the string entirely like "{host}:{port}" (and do not read the .format part) as opposed to "{}:{}"

>> Its just common sense to make things readable.

Agree.

>> Let's try to stick to making this criticism when its more appropriate.
 
Nod.  So, will avoid it for simpler formats. But when it really is necessary, let's enforce it.